platform/ios/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
+* Fixed crash in `-[MGLStyle localizeLabelsIntoLocale:]` on iOS 9.3 (attempting to access a property that was introduced in iOS 10.0) ([#12123](https://github.com/mapbox/mapbox-gl-native/pull/12123))
```

Comment:
We should match the entry that went into the 4.1.0 changelog

Reply:
For this cp I'm just adding leaving the related blurb. In the bump pr I will make this as it should be.